### ISSUE TITLE
Funny bug UTF-8

### ISSUE THREAD

**Issue by NuVivo314** [modoboa/modoboa-sievefilters#10]
Hello,

I'm discover a small bug. Il modoboa is config in French language, sievefilters don't work!

```
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.sites',
 'django.contrib.staticfiles',
 'reversion',
 'rest_framework.authtoken',
 'modoboa',
 'modoboa.core',
 'modoboa.lib',
 'modoboa.admin',
 'modoboa.relaydomains',
 'modoboa.limits',
 'modoboa_amavis',
 'modoboa_pdfcredentials',
 'modoboa_postfix_autoreply',
 'modoboa_sievefilters',
 'modoboa_stats',
 'modoboa_webmail',
 'modoboa_radicale')
Installed Middleware:
('x_forwarded_for.middleware.XForwardedForMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'modoboa.lib.middleware.AjaxLoginRedirect',
 'modoboa.lib.middleware.CommonExceptionCatcher',
 'modoboa.lib.middleware.RequestCatcherMiddleware')

Traceback:
File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  132.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/utils/decorators.py" in inner
  145.                     return func(*args, **kwargs)
File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/utils/decorators.py" in inner
  145.                     return func(*args, **kwargs)
File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  22.                 return view_func(request, *args, **kwargs)
File "/srv/modoboa/env/local/lib/python2.7/site-packages/modoboa/admin/lib.py" in wrapped_f
  29.                 return f(request, *args, **kwargs)
File "/srv/modoboa/env/local/lib/python2.7/site-packages/modoboa_sievefilters/views.py" in getfs
  103.             _("Actions"), fset_menu(editormode, name)

Exception Type: UnicodeEncodeError at /sfilters/back/
Exception Value: 'ascii' codec can't encode character u'\xe9' in position 731: ordinal not in range(128)
```

In english, it's work fine.

**Comment by tonioo** [comment]
Hi, have you tried to upgrade the sievelib package ?

**Comment by tonioo** [comment]
Fixed by #11
